src/Bindings/LuaState.cpp

```diff
@@ -39,6 +43,10 @@ extern "C"
 
 const cLuaState::cRet cLuaState::Return = {};
 
+/** Each Lua state stores a pointer to its creating cLuaState in Lua globals, under this name.
+This way any cLuaState can reference the main cLuaState's TrackedCallbacks, mutex etc. */
+static const char * g_CanonLuaStateGlobalName = "__Cuberite_CanonLuaState";
```

Comment:
We also have `_CuberiteInternal_PluginName`, but that was/is exported by accident. I assume this one isn't exported?